e2e/test/mutate-specific-lines/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "mutate-specific-lines",
+  "description": "A test project for only running specific mutants, rather than all mutants",
+  "scripts": {
+    "test": "stryker run -m src/index.js:3:18:3:26,src/index.js:4:18:4:25",
+    "test:unit": "jest",
+    "posttest": "mocha --require ../../tasks/ts-node-register.js verify/verify.ts"
+  },
+  "devDependencies": {
+    "@types/jest": "~25.2.1",
```

Comment:
I don't know at this point. Will they be pulled in automatically? I assumed I needed them because this specific test uses jest, but perhaps I don't understand what dependencies are pulled in where in the e2e tests

Reply:
The e2e parent project itself has a big list of dependencies. Specific e2e projects only need their own dependencies to test special cases. For example angular-project, vue-cli project, etc. They generally don't play nice with "hoisted" dependencies (sigh...).

For e2e tests testing stuff within Stryker itself (coverage analysis, reporters, and I think this mutant range feature as well) you can probably get away with a simple project that relies on the parent's dependencies.